core/server/services/webhooks/trigger.js

```diff
@@ -23,7 +24,8 @@ function makeRequests(webhooksCollection, payload, options) {
             body: reqPayload,
             headers: {
                 'Content-Length': Buffer.byteLength(reqPayload),
-                'Content-Type': 'application/json'
+                'Content-Type': 'application/json',
+                'X-Ghost-Secret': webhookSecret
```

Comment:
I think we should be sending some kind of MAC here so as to not expose the secret over the network